temperature/router.py

```diff
@@ -0,0 +1,39 @@
+from sqlalchemy.orm import Session
+from fastapi import APIRouter, Depends, HTTPException
+
+from . import schemas, crud
+from dependencies import get_db
+
+
+router = APIRouter()
+
+
+@router.get("/temperature", response_model=list[schemas.TemperatureDefault])
+def get_all_temperatures(db: Session = Depends(get_db)):
+    return crud.get_all_temperatures(db)
+
+
+@router.get("/city/temperatures", response_model=list[schemas.TemperatureDefault])
+def get_temperatures_by_city_id(city_id: int, db: Session = Depends(get_db)):
+    db_city_id = crud.get_city_id(db, city_id)
+    if not db_city_id:
+        raise HTTPException(
+            status_code=400,
+            detail=f"city_id={city_id} was not found"
+        )
+    else:
+        return crud.get_temperature_by_city_id(db, city_id)
+
+
+@router.post("/temperature/", response_model=schemas.TemperatureDefault,

```

Comment:
This is wrong endpoint, check the task description.

Reply:
fixed